src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/ReadmePreviewViewModel.cs

```diff
@@ -73,6 +72,7 @@ private static bool CanRenderLocalReadme(ItemFilter filter)
 
         private async Task LoadReadmeAsync(CancellationToken cancellationToken)
         {
+            ReadmeMarkdown = Resources.Text_Loading;
```

Comment:
As mentioned in my previous review's comment, I think we should represent loading or busy state in the ViewModel and decide how that should be presented in the View.